src/index.html

```diff
@@ -1,12 +1,344 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="colored moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-&-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="container">
+      <div class="card" data-qa="card">
+        <picture class="card__picture">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__image"
+          >
+        </picture>
+
+        <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <span class="card__subtitle">Product code: 195434</span>
```

Comment:
```suggestion
        <p class="card__subtitle">Product code: 195434</p>
```
`p` is semantic tag, `span` is not - use `p` for text